### ISSUE TITLE
traefikCRD with lets-encrypt makes traefik sad

### ISSUE THREAD

**Issue by jbergler** [authelia/chartrepo#5]
Hey there.

Thanks for working on the chart! I noticed in trying to use it that the current template for the `IngressRoute` always sets a secret name, even though this isn't required when using a resolver.

This results in my Traefik logs being spammed with 
```
level=error msg="Error configuring TLS: secret kube-system/authelia-tls does not exist" providerName=kubernetescrd ingress=authelia namespace=kube-system
```

I'm not super sure, but I suspect it's this line here, that either needs to disappear when a resolver is specified or be somehow configurable so it doesn't appear.

https://github.com/authelia/chartrepo/blob/1dd3a04a1406c5f7c5659765ab5b25a1323b3a43/charts/authelia/templates/traefikCRD/ingressRoute.yaml#L43

**Comment by jbergler** [comment]
And to sort of answer my own question I tried setting 
```
ingress:
  tls:
    secret: ""
  traefikCRD:
    tls:
      certResolver: lets-encrypt
```

And that shut Traefik up, so I guess that's something...

**Comment by james-d-elliott** [comment]
That suggestion makes perfect sense. We can just add a conditional. I actually don't use TLS with traefik (external termination with haproxy). So it wasn't something I explicitly tested. The secret should be there if someone statically configured a TLS cert/key secret, I think.

**Comment by james-d-elliott** [comment]
Fixed in 17bc7c107348c9e356da69db736dab363a56ed54
